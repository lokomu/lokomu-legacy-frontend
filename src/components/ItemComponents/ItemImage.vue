<template>
  <img
    :src="image"
    :class="{
      'w-24 h-24 object-cover rounded-full': profile,
    }"
    alt="Image"
  >
</template>

<script setup lang="ts">
defineProps<{
  image: string;
  profile?: boolean;
}>();
</script>
