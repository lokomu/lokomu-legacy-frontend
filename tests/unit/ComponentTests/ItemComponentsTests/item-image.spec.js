import { mount } from "@vue/test-utils";
import ItemImage from "@/components/ItemComponents/ItemImage.vue";
import { describe, expect, it, beforeEach } from "vitest";

describe("FormImageDisplay component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(ItemImage);
  });

  it("renders correctly", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("is instantiated", () => {
    expect(wrapper.exists()).toBeTruthy();
  });
});
