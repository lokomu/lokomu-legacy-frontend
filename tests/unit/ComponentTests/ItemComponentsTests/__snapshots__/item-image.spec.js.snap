// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FormImageDisplay component > renders correctly 1`] = `
<img
  alt="Image"
  class=""
/>
`;
